src/main/java/com/alipay/oceanbase/rpc/protocol/payload/impl/execute/ObTableOperation.java

```diff
@@ -127,14 +127,15 @@ public static ObTableOperation getInstance(ObTableOperationType type, Object[] r
         obTableOperation.setOperationType(type);
         ObITableEntity entity = new ObTableEntity();
         obTableOperation.setEntity(entity);
-        for (int i = 0; i < rowKeys.length; i++) {
-            Object rowkey = rowKeys[i];
-            ObObjMeta rowkeyMeta = ObObjType.defaultObjMeta(rowkey);
-
-            ObObj obj = new ObObj();
-            obj.setMeta(rowkeyMeta);
-            obj.setValue(rowkey);
-            entity.addRowKeyValue(obj);
+        if (rowKeys != null) {
```

Comment:
What about this situation: c1 and c2  both are primary keys and c1 is an auto-inc key.

Reply:
In the above situation, if the rowkey array is empty, obkv will report an error, but it will report the error on the server side. This place only determines whether the input rowkey array is empty. The client sends this input to the server. The specific behavior is determined by the server.